.github/workflows/integration-test-smoke-suite.yaml

```diff
@@ -51,3 +51,7 @@ jobs:
       with:
         name: ceph-smoke-suite-artifact
         path: /home/runner/work/rook/rook/tests/integration/_output/tests/
+
+    - name: setup tmate session for debugging
+      if: failure() && contains(github.event.pull_request.labels.*.name, 'debug-smoke-suite')
```

Comment:
Once we run on multiple versions of K8s, we will still use the same label across the versions, right? At least I hope the labels won't multiply with the versions. :)

Reply:
Yes, we will but we can add more labels and pass the matrix variable in the `contains` so we don't do it on all jobs.